packages/next/build/index.ts

```diff
@@ -89,6 +90,15 @@ export default async function build(dir: string, conf = null): Promise<void> {
   const { target } = config
   const buildId = await generateBuildId(config.generateBuildId, nanoid)
   const distDir = path.join(dir, config.distDir)
+  const rewrites = []
+  const redirects = []
+
+  if (typeof config.rewrites === 'function') {
```

Comment:
nit: can we swap the order here to reflect how it's applied in prod

Reply:
Updated